test/packages/integration_benchmarks/_dev/benchmark/system/tcp-benchmark/config.yml

```diff
@@ -0,0 +1,40 @@
+- name: IP
```

Comment:
Should this file `config.yml` and `fields.yml` be added to the specification ?

Are those files required if the folder is present?

Reply:
The folder is supposed to be a placeholder for any resources required by the benchmark, the idea of not making it based on the benchmark name was based on the idea that different benchmarks may want to use the same resources. So both the folder names and its contents were just not enforced any particular structure.

Reply:
Would that mean that there could be any file inside those folders ?

Reply:
even though I see this not being so common, yes it could be any number of them

Reply:
Talked offline, a new entry will be added to define resource folders explicitly. And those folder could contain any resource.